playbooks/common/openshift-node/service.yml

```diff
@@ -10,9 +10,9 @@
     add_host: name={{ item }} groups=g_service_nodes
     with_items: oo_host_group_exp | default([])
 
-- name: Change openshift-node state on node instance(s)
+- name: Change state on node instance(s)
   hosts: g_service_nodes
   connection: ssh
   gather_facts: no
   tasks:
-    - service: name=openshift-node state="{{ new_cluster_state }}"
+    - service: name={{ product_type }}-node state="{{ new_cluster_state }}"
```

Comment:
On that note, I think I would rather target this PR for after the naming changes happen, otherwise we are looking at 2 huge invasive PRs, one for getting he initial support and then another for managing the naming changes.